setup.py

```diff
@@ -181,7 +181,7 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
     'atlasclient>=0.1.2',
 ]
 azure = [
-    'azure-batch>=8.0.0',
+    'azure-batch>=8.0.0,<10.0.0',
```

Comment:
You can likely remove that limitation as well.

Reply:
I had to add another restriction. [pyjwt](https://pypi.org/project/PyJWT/#history) was updated a few hours ago and is causing the problem.